exo-create/helpers/helpers.go

```diff
@@ -0,0 +1,67 @@
+package helpers
+
+import (
+	"io/ioutil"
+	"os"
+	"path"
+
+	"github.com/pkg/errors"
+)
+
+const projectJsonContent = `
+{
+  "AppName": "my-app",
+  "ExocomVersion": "0.22.1",
+  "AppVersion": "0.0.1",
+  "AppDescription": ""
+}
+`
+
+const applicationYmlContent = `name: {{AppName}}
```

Comment:
Auhh okay. Then lets remove the newline we are currently adding from the project.json. Than go ahead and ship if the build stays green